src/OrchardCore.Modules/OrchardCore.Lists/Views/ListPartFeed.Edit.cshtml

```diff
@@ -18,18 +18,3 @@
     <input asp-for="FeedItemsCount" class="form-control" />
     <span class="hint">@T["The number of elements to display in the feed."]</span>
 </div>
-
-<script at="Foot">
```

Comment:
I'm not sure why its delete!! no need to show the feed URL if the feature is disabled

Reply:
The feed still works when the "disable rss feed" property is checked. Technically we could have another property to actually disable the feed, this one will just remove the link in the HTML, but hitting the url will still work. I was not sure about the change to apply on the JS so for now the safe way was to just remove the hiding logic.